src/pages/AboutUser/AboutUser.jsx

```diff
@@ -62,6 +69,7 @@ const AboutUser = () => {
           </div>
         </div>
       </div>
+      <LeafletMap  cordinates={arrofCord.length===0 ? defaultCord : arrofCord} name={name}/>
```

Comment:
use `arrofCord.length ? affordCord : defaultCord` instead of `arrofCord.length===0 ? defaultCord : arrofCord`